<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>287</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CLAS12 DC WIRE MAP</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QStackedWidget" name="left_stacked_widget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="explorer_holder"/>
      <widget class="QWidget" name="chooser_holder"/>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="Runset" native="true">
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="horizontalWidget" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Run Number:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="rundisplay">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wiremap_holder" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>wiremap_holder</zorder>
      <zorder>horizontalWidget</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionExplorer"/>
    <addaction name="actionChooser"/>
   </widget>
   <widget class="QMenu" name="menuSet_Run">
    <property name="title">
     <string>Set Run </string>
    </property>
    <addaction name="actionEdit_Run_Number"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuSet_Run"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionComponentChooser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Component Chooser</string>
   </property>
  </action>
  <action name="actionComponentExplorer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Component Explorer</string>
   </property>
  </action>
  <action name="actionExplorer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Explorer</string>
   </property>
  </action>
  <action name="actionChooser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Chooser</string>
   </property>
  </action>
  <action name="actionSet_Run_Number">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Set Run Number</string>
   </property>
  </action>
  <action name="actionEdit_Run_Number">
   <property name="text">
    <string>Edit Run Number</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExplorer</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>setModeExplorer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChooser</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>setModeChooser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEdit_Run_Number</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>setRunDialogue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSet_Run_Number</sender>
   <signal>triggered()</signal>
   <receiver>Runset</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setModeChooser()</slot>
  <slot>setModeExplorer()</slot>
  <slot>loadRun(int)</slot>
  <slot>setRunDialogue()</slot>
 </slots>
</ui>
